### ISSUE TITLE
Crashes on Quest 2 Godot 4.2 Mono

### ISSUE THREAD

**Issue by ColtonDaiber** [GodotVR/godot_openxr_vendors#58]
In Godot 4.2 release candidate 1 mono and Godot 4.2 beta 5 mono

TLDR: the game initially launches like a 2d app, then the game goes into vr mode for 1-2 frames. finally the takes the user back to the quest's home with a bugged ui 

The app crashes with the plugin's Quest option enabled in the export menu
When launching the app a window appears like the game is a 2d application, and then after a second the window disappears. Sometimes you can notice one frame flash on the headset and it looks like the game actually started in vr, but then the headset goes back to the home environment, but the ui is bugged. The menu isnt there and the menu buttons dont work.

Without the quest option checked
the app launched with a window like its trying to run a normal 2d application, and theres an error about openxr failing to start.

The reason i am using godot 4.2 is because the previous versions of godot 4 do not support exporting to android with godot mono and I am planning on using c# for my code
